### DISCUSSION TITLE
2 canvases OR 1?

### DISCUSSION THREAD

**Discussion by spantula** [fabricjs/fabric.js#8784] [Q&A]
Friends, We are using Fabricjs to generate two-sided employee id cards in Angular. 
We need help with
![card](https://user-images.githubusercontent.com/17843090/225212816-cf8bb270-be93-4ed9-af23-cf8af4d94ab9.PNG)
 the following

We have a page in which we have two canvas displaying the front and back of an employee id card. Please see the attached image to better understand.
When we add text to the canvas it is printed on both sides. I would like to edit the front and back independently of each other. Can we have two canvas instances on one page and be able to control them independently?

**Comment by ShaMan123** [comment]
Of course you can.
But IMO should be one canvas with 2 clipped objects. That way you the user can interact between the 2 with ease (e.g. DnD, copy paste)
